docs/manifest.json

```diff
@@ -40,8 +40,14 @@
       ["Restrictions", "/authentication/configuration/restrictions"],
       ["SAML at Clerk", "/authentication/saml-at-clerk"],
       "# Prebuilt Components",
-      ["<SignIn />", "https://clerk.com/docs/components/authentication/sign-in"],
-      ["<SignUp />", "https://clerk.com/docs/components/authentication/sign-up"],
+      [
```

Comment:
@BRKalow there are a lot of formatting changes to the manifest—could you remove these to keep this PR to just the error boundary?